src/annepro2.rs

```diff
@@ -50,16 +54,18 @@ pub enum AP2FlashError {
     OtherError,
 }
 
-pub fn flash_firmware<R: std::io::Read>(target: AP2Target, base: u32, file: &mut R, interface: i32, boot: bool) -> std::result::Result<(), AP2FlashError> {
+pub fn flash_firmware<R: std::io::Read>(target: AP2Target, base: u32, file: &mut R, boot: bool) -> std::result::Result<(), AP2FlashError> {
     match HidApi::new() {
         Ok(api) => {
             for dev in api.device_list() {
-                println!("HID Dev: {:04x}:{:04x} if: {} {:?}", dev.vendor_id(), dev.product_id(), dev.interface_number(), dev.product_string());
+                println!("HID Dev: {:04x}:{:04x} usage #: {:02x} usage_page #: {:04x} {:?}", dev.vendor_id(), dev.product_id(),
+                dev.usage(), dev.usage_page(), dev.product_string());
             }
             let dev = api.device_list().find(|dev| {
                 dev.vendor_id() == ANNEPRO2_VID &&
```

Comment:
I don't think this should replace the old VID + PID pair. I think we should still keep that. Since their VID is not their own. I think having PID check is good.

Reply:
What do you mean? This line still prints out pid and vid

Reply:
Sorry it was late brain wasn't really functioning. But thanks for the change.